xplique/plots/metrics.py

```diff
@@ -0,0 +1,158 @@
+"""
+Plots for metric
+"""
+
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+from ..types import Optional, Dict, Any
+
+
+def barplot(
+        scores: Dict[str, Dict[str, float]],
+        sort_metric: str = None,
+        ascending: bool = False,
+        title: str = "",
+        filepath: str = None,
+        methods_colors: Optional[Dict[str, Any]] = None
+):
+    """
+    Make a bar-plot regrouping the score of each method for different metrics
```

Comment:
Sentence1 & 2 are a bit redundant, I would stick with 
` Make a bar chart gathering the score of each method for the different metrics used.` Also I think metric may be replaced with loss, but in that case we also need to replace the parameter name. For the moment it's consistent ^^